apps/coscrad-frontend/README.md

```diff
@@ -39,6 +39,12 @@ To run the front-end and back-end concurrently from a single command, run
 
 Note that you will need to have the backend configured in the `api` application, including a working instance of ArangoDB that is linked in your `.env.` See the [docs for `api`](../api/README.md) for more information.
 
+### Redux Plugin
+
+To inspect the front-end data stored in Redux, use Redux DevTools in [Chrome](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd), [Edge](https://microsoftedge.microsoft.com/addons/detail/redux-devtools/nnkgneoiohoecpdiaponcejilbhhikei), or [Firefox](https://addons.mozilla.org/en-US/firefox/addon/reduxdevtools/).
```

Comment:
To inspect the in-memory front-end state stored in Redux...

Reply:
updated